PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/PBMBidRequester.m

```diff
@@ -157,6 +159,8 @@ - (void)makeRequestWithCompletion:(void (^)(BidResponse *, NSError *))completion
         }
         
         completion(bidResponse, trasformationError);
+        [Prebid.shared callEventDelegate_prebidBidRequestDidFinishWithRequestData:rtbRequestData 
```

Comment:
I have a concern here that this call can lead to blocking behavior due to the client's implementation. Can we call this delegate asynchronously?  

The async call can be implemented inside `callEventDelegate_prebidBidRequestDidFinishWithRequestData`. In this case the method name should reflect the async behaviour. For instance, `callEventDelegateAsync_prebidBidRequestDidFinishWithRequestData`. 

Open question: on which thread should methods of `PrebidEventDelegate` be called? It shouldn't be a main thread.